new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -199,49 +258,451 @@ public void testDeleteOperatorButNotDomain() {
    */
-  @AfterAll
-  public void tearDownAll() {
-    // uninstall NGINX release
-    if (nginxHelmParams != null) {
-      assertThat(uninstallNginx(nginxHelmParams))
-          .as("Test uninstallNginx returns true")
-          .withFailMessage("uninstallNginx() did not return true")
-          .isTrue();
+  @Test
+  @DisplayName("Create operator and domain, then delete operator and"
+      + " create again operator and verify it can still manage domain")
+  @Slow
+  @MustNotRunInParallel
+  public void testCreateDeleteCreateOperatorButNotDomain() {
+    String opReleaseName = OPERATOR_RELEASE_NAME;
+    HelmParams op1HelmParams = new HelmParams().releaseName(opReleaseName)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+    String opServiceAccount = opNamespace + "-sa";
+    try {
+      // install operator
+
+      HelmParams opHelmParams = installAndVerifyOperator(opNamespace, opServiceAccount, true,
+          0, op1HelmParams, domain1Namespace);
+      assertNotNull(opHelmParams, "Can't install operator");
+
+      if (!isDomain1Running) {
+        logger.info("Installing and verifying domain");
+        assertTrue(createVerifyDomain(domain1Namespace, domain1Uid),
+            "can't start or verify domain in namespace " + domain1Namespace);
+        isDomain1Running = true;
+      }
+      // delete operator
+      logger.info("Uninstalling operator");
+      uninstallOperator(opHelmParams);
+
+      //install second time
+      opHelmParams = installOperatorHelmChart(opNamespace, opServiceAccount, false, true, false,
+          null,"deployed", 0, opHelmParams, false, domain1Namespace);
+
+      assertNotNull(opHelmParams, "Can't install operator");
+      int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
+      //check if can still manage domain1
+      assertTrue(scaleDomain(domain1Namespace, domain1Uid,2,1,
+          opNamespace, opServiceAccount, externalRestHttpsPort), "Domain1 " + domain1Namespace + " scaling failed");
+    } finally {
+      deleteSecret("ocir-secret",opNamespace);
+      cleanUpSA(opNamespace);
+      uninstallOperator(op1HelmParams);
+    }
+  }
+
+  /**
+   * Create operator and verify it is deployed successfully
+   * Create domain1 and verify the domain is started
+   * Upgrade the operator domainNamespaces to include namespace for domain2
+   * Verify both domains are managed by the operator by making a REST API call
+   * Call helm upgrade to remove the first domain from operator domainNamespaces
+   * Verify it can't be managed by operator anymore.
+   * Test fails when an operator fails to manage the domains as expected
+   */
+  @Test
+  @DisplayName("Create domain1, managed by operator and domain2, upgrade operator for domain2,"
+      + "delete domain1 , verify management for domain2 and no access to domain1")
+  @Slow
+  @MustNotRunInParallel
+  public void testAddRemoveDomainUpdateOperatorHC() throws Exception {
```

Comment:
What is HC ?  The method name should be self explanatory as far as possible
How about testAddRemoveDomainNameSpacesOnOperator()

Reply:
HC- helm chart :) , fixed the method name